beacon_node/network/src/beacon_processor/work_reprocessing_queue.rs

```diff
@@ -770,6 +845,32 @@ impl<T: BeaconChainTypes> ReprocessQueue<T> {
                     }
                 }
             }
+            InboundEvent::ReadyBackfillSync(queued_backfill_batch) => {
+                let seconds_from_slot_start = slot_clock
+                    .seconds_from_current_slot_start()
+                    .map_or("unknown".to_string(), |duration| {
+                        duration.as_secs().to_string()
+                    });
+
+                debug!(
+                    log,
+                    "Sending scheduled backfill work to Beacon Processor at {} seconds from current slot start",
+                    seconds_from_slot_start
+                );
+
+                if self
+                    .ready_work_tx
+                    .try_send(ReadyWork::BackfillSync(queued_backfill_batch.clone()))
+                    .is_err()
+                {
+                    error!(
+                        log,
+                        "Failed to send scheduled backfill work. Sending it back to queue.";
+                    );
```

Comment:
```suggestion
                    error!(
                        log,
                        "Failed to send scheduled backfill work";
                        "info" => "sending work back to queue"
                    );
```

This is along the same lines as my previous suggestion.